Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM mhart/alpine-node:latest
+RUN apk add --no-cache git file
+RUN git clone https://github.com/CapacitorSet/box-js /usr/share/boxjs;\
+cd /usr/share/boxjs; npm install;
```

Comment:
`WORKDIR /usr/share/boxjs`; also, use `npm install --production` to avoid installing unused dependencies like Mocha and ESLint.